web/client/translations/data.en-US.json

```diff
@@ -2225,6 +2227,7 @@
             "emptyRuleEditor": "There are not rules for this style. Add a new rule by clicking on the desidered symbolizer type in the toolbar",
             "classificationError": "Service unavailable. Currently it is not possible to create a classification",
             "classificationRasterError": "Service unavailable. Currently it is not possible to create a raster classification",
+            "classificationUniqueIntervalError": "Style classes produced by the classification is greater than the intervals for unique specified",
```

Comment:
Maximum intervals limits exceeded! You are trying to generate an unique interval classification with an attribute that contains more than {intervalsForUnique} different classes. Unique Interval classification should be applied to attributes with a limited set of intervals to allow a better readability of the style. You could try different method for the selected attribute.

We could use the msgParams for {intervalsForUnique}

please add translation also for other languages